hub/src/test/java/ch/prevo/open/hub/HubServiceTest.java

```diff
@@ -44,4 +50,32 @@ public void matchAndNotify() throws Exception {
         assertEquals(exit.getRetirementFundUid(), notifiedMatches.get(0).getPreviousRetirementFundUid());
         assertEquals(entry.getRetirementFundUid(), notifiedMatches.get(0).getNewRetirementFundUid());
     }
+
+    @Test
+    public void matchAndNotifySeveralTerminationsForSingleCommencement() {
```

Comment:
This test does not really test that they're really notified. It only tests that they're matching. Because in the `NodeCaller` there is a check that notifications are only sent once.